modules/trusted-firmware-m/Kconfig.tfm

```diff
@@ -177,6 +177,28 @@ config TFM_PARTITION_PLATFORM_CUSTOM_REBOOT
 	  Instead the application will have to override the weak ARM
 	  implementation of sys_arch_reset().
 
+config TFM_DUMMY_PROVISIONING
+	bool "Provision with dummy values. NOT to be used in production"
+	select TFM_INITIAL_ATTESTATION_KEY
+	default n if BOARD_B_U585I_IOT02A
+	default y
+	help
+	  If this option is enabled (as it is by default), a set of dummy
+	  keys / data will be provisioned. The dummy IAK matches the IAK tested
+	  by the TF-M tests, and the dummy bl2 ROTPKs match the dummy bl2 keys
+	  used by default.
+	  This option MUST not be used in production hardware, as the keys are
+	  insecure.
+
+config TFM_INITIAL_ATTESTATION_KEY
+	bool
+	help
+	  Hidden option to mark that the TF-M platform has an initial
+	  attestation key, which is a requirement for the Initial Attestation
+	  partition.
+	  A TF-M platform that supports the usage of Initial Attestation Key
+	  shall enable this configuration.
```

Comment:
Is it a symbol that indicates whether the HW/SDK supports such a key. Of a symbol that indicates whether it is present in the current build?

I feel these two sentences are communicating different things.

Reply:
Whether the HW / SDK supports this key. Given that self-provisioning of the IAK is not something that should be done according PSA, this means that it is not really part of the build, but part of the whole "system" of this platforms build.
Like a provisioning image for example.

Reply:
Removed second sentence.